test_unified_dimensions.py

```
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+"""
+Test script to verify embedding dimensions are working correctly.
+"""
+import asyncio
+import logging
+from unified_ingestion import UnifiedIngestionPipeline, IngestionConfig
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+async def test_embedding_dimensions():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_embedding_dimensions`

Add return type annotation: `bool | None`

(ANN201)